.rubocop.yml

```diff
@@ -12,6 +12,10 @@ AllCops:
     - 'vendor/**/*'
   TargetRubyVersion: 2.4
 
+Packaging/RelativeRequireToLib:
+  Include:
+    - spec/**/*.rb
```

Comment:
also tests/, test/ specs/

Reply:
That will depend on each project.
Those who have `tests` directory, should replace this line with `tests/**/*.rb` in their `.rubocop.yml`.

This `.rubocop.yml` describes the structure for **this** project. Having `tests/`, `test/`, or `specs/` here wouldn't make sense.

Reply:
ah right, I confused with the default settings